docs(grammar): fix using grammarly (#759)
EddieHubCommunity/LinkFree: README.md

> A note for Hacktoberfest Participants:
>

> > Pull requests which add or edit your information in a `data/${yourname}.json` file will NOT count for Hacktoberfest.

Reviewer: Be counted works better here

Author: It is incorrect use of passive voice tho.

Reviewer: I don’t think it’s wrong. The vote will be counted by some other subject so will be sounds better. Remember grammarly is a computer program that can’t always detect nuances so it’s not always correct either. As a native English speaker sometimes things just sound wrong to the ear, like it’s not natural and for me this is one of those times. In this case it’s not a massive issue but be wary in believing everything grammarly tells you ;-)

Author: ```suggestion
> Pull requests which add or edit your information in a `data/${yourname}.json` file will NOT be counted for Hacktoberfest.
```

}
```

Your URL will be `http://linkfree.eddiehub.org/<yourusername>`

> For example <http://linkfree.eddiehub.org/eddiejaoude>\

Reviewer 2: ```suggestion
For example: <http://linkfree.eddiehub.org/eddiejaoude>\
```

## 🛡️ License

> Linkfree is licensed under the MIT License - see the [`LICENSE`](LICENSE) file for details.

Reviewer 2: ```suggestion
LinkFree is licensed under the MIT License - see the [`LICENSE`](LICENSE) file for details.
```
